Add some motivation to the Beaker docs. (#32)
beaker/docs: README.md

# User Guide

[Beaker](https://beaker.org) is a collaborative platform for rapid, reproducible research.  Beaker
frees you from manually managing your machines, allowing you to seamlessly:

> * to move experiments between the cloud and on-premise with a single line of code
> * script hundreds of related experiments to run in parallel in the cloud or as space allows
> on-premise
> * view your experiments progress and results using the web UI at http://www.beaker.org
> * create an isolated interactive session for development and debugging

Reviewer: Gramar nits ...
```suggestion
* move experiments between the cloud and on-premise with a single line of config.
* script hundreds of related experiments to run in parallel.
* view your experiments' progress and results using the [web UI](http://www.beaker.org).
* create an isolated interactive session for development and debugging.
```
